DTS-STN/canadian-dental-care-plan
# Changing requested mocks in preparation for demo

### Description
As per title - special request from the big boss.

- Change the ref number in the online app to `2476124092174`
- Change the mock to have a single determination letter (`775170001`) with a date of today

New PDF for Determination letter should come soon... in a future PR?

### Screenshots
**Reference number:**
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/88568202-6109-4a08-8a2c-a52305cecc48)

**Determination letter:**
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/3d758386-d741-46d1-a31f-114f049990d5)

### Checklist

### Test Instructions
Run application and verify that the reference number is the same as the description after submitting a CDCP application.
Navigate to `/en/letters` and confirm there is one Determination letter.

### Additional Notes
No time to spin up my own environment for this - plan is to use dev instead.
